### ISSUE TITLE
Error originated by #805

### ISSUE THREAD

**Issue by PeterVoronov** [iobroker-community-adapters/ioBroker.telegram#809]
**Describe the bug**  
Bug made by me due to wrong order of code strings. 

**To Reproduce**  
Install latest version and wait for any error in telegram
Will be produced 
```
Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-04-03 15:35:55.132  -  [31merror [39m: host.iot-main Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2024-04-03 15:35:55.132  -  [31merror [39m: host.iot-main Caught by controller[0]: TypeError: Cannot read properties of null (reading 'chat_id')
2024-04-03 15:35:55.132  -  [31merror [39m: host.iot-main Caught by controller[0]:     at /opt/iobroker/node_modules/iobroker.telegram/main.js:801:42
```
"Type error: Cannot read properties of null (reading 'chat_id') 

**Expected behavior**  
No this error

**Screenshots & Logfiles**  
See above

Fixed by #808

**Comment by PeterVoronov** [comment]
Excuse me, error due to wrong copy code from test system...

**Comment by mcm1957** [comment]
fixed with 3.2.1

**Comment by PeterVoronov** [comment]
> fixed with 3.2.1

Thanks!
